DisplaySettings/DisplaySettings.json

```diff
@@ -1342,8 +1404,8 @@
                 ]
             }
         },
-        "getVolumeLeveller": {
-            "summary": "(Version 2) Returns the current Volume Leveller setting",
+        "getVolumeLeveller(v2)": {
```

Comment:
Remove (v2) from the method signature. Application calls that include (v2) in the method signature will fail.

Reply:
As discussed keeping only version2 methods and removed (v2) from method.